js_modules/dagster-ui/packages/ui-core/src/runs/useRunsForTimeline.tsx

```diff
@@ -23,99 +25,177 @@ import {repoAddressAsHumanString} from '../workspace/repoAddressAsString';
 import {RepoAddress} from '../workspace/types';
 import {workspacePipelinePath} from '../workspace/workspacePath';
 
+const BUCKET_SIZE = 3600 * 1000;
```

Comment:
nit: nice to have unit  in name and comment explaining what the value is `BUCKET_SIZE_MS = 3600 * 1000 // 1hr`